tests/testthat/test-lyt-tabulation.R

```diff
@@ -365,7 +364,7 @@ test_that("error localization works", {
 })
 
 
-test_that("cfun args", {
+test_that("cfun works with first arg df", {
```

Comment:
Please consolidate this and the test below into a single test set (under the old 'cfun args' name)